src/main/java/de/maxhenkel/audioplayer/command/AudioPlayerCommands.java

```diff
@@ -378,7 +378,7 @@ private static LiteralArgumentBuilder<CommandSourceStack> applyCommand(LiteralAr
                                 })));
     }
 
-    private static LiteralArgumentBuilder<CommandSourceStack> bulkApplyCommand(LiteralArgumentBuilder<CommandSourceStack> builder, Predicate<ItemStack> validator, String itemTypeName) {
```

Comment:
The `itemTypeName` would be useful to keep, so that we can support other containers in the future.
The `validator` wouldn't make sense to keep if you don't reuse the `itemTypeName`.